.github/workflows/run_psi4_test.yml

```diff
@@ -2,13 +2,10 @@ name: psi4 test
 
 on: [pull_request]
 
-env:
-  NO_PYSCF: 1
-
 jobs:
   build:
 
-    runs-on: ubuntu-latest
+    runs-on: windows-latest
```

Comment:
I like that we are putting windows into the picture, but now we're a bit all over the place.

- why this change ?
- does that mean we have tests running on ubuntu with pypi , but windows with psi4 ? Do we believe that's what our users are mostly using ?

As a result of this change, are we no longer testing a configuration that we should be testing ?

Reply:
I was mostly using it to test whether we need to direct people to use Windows Linux Subsystem. I can change it back. But the tests are now passing and it appears people can use Tangelo in Windows (native) with Psi4.

Reply:
Great information. Let's stick to Ubuntu now and make the matter of extending things to Windows / Mac / whatever the scope of a subsequent PR, to do things in a smart way.